layouts/layout.js

```diff
@@ -151,7 +151,7 @@ const Layout = ({
           </div>
         )}
       </article>
-      <div className="flex justify-between font-medium text-gray-500 dark:text-gray-400">
+      <div className='flex justify-between font-medium text-gray-500 dark:text-gray-400 my-5'>
```

Comment:
Could you wrap the value with double quotes? Since we all use double quotes for attributes.